vizro-core/src/vizro/actions/_parameter_action.py

```diff
@@ -23,10 +24,10 @@ def _parameter(targets: List[str], **inputs: Dict[str, Any]) -> Dict[str, Any]:
     Returns:
         Dict mapping target component ids to modified charts/components e.g. {'my_scatter': Figure({})}
     """
-    targets = [target.split(".")[0] for target in targets]
+    target_ids: List[ModelID] = [target.split(".")[0] for target in targets]  # type: ignore[misc]
 
     return _get_modified_page_charts(
-        targets=targets,
+        targets=target_ids,
```

Comment:
Should we revert these changes here or refactor them in other predefined actions too?

Reply:
Specifically the change on line 30?

Reply:
I thought you should leave the name "targets" instead of this name "target_ids", but I don't mind this either, so it doesn't matter and leave it like this.